src/app/Library/CrudPanel/Traits/Create.php

```diff
@@ -112,6 +112,10 @@ public function syncPivot($model, $data)
             if (isset($field['pivot']) && $field['pivot']) {
                 $values = isset($data[$field['name']]) ? $data[$field['name']] : [];
 
+                //we override the values if the field is repeatable and pivot == true
+                if ($field['type'] == 'repeatable') {
```

Comment:
We just talked about this - you had the great idea to do things here NOT if the field type is repeatable, but if the:
- field has `pivot==true`
- field has `fields` array (exists and not empty)
- (maybe even) $field['fields'] contains a field with the same name as the parent field (the relationship)

Then we can basically be sure this is what the developer wants.

Reply:
Just did the changes.